bundles/org.openhab.automation.jsscripting/src/main/java/org/openhab/automation/jsscripting/internal/OpenhabGraalJSScriptEngine.java

```diff
@@ -70,7 +70,8 @@
     private static final Path NODE_DIR = Paths.get("node_modules");
 
     // shared lock object for synchronization of multi-thread access
-    private final Object lock = new Object();
+    public final Object lock = new Object();
```

Comment:
Why public? Shouldn't this remain private?

Reply:
Yes.
Seems like code completion did not what I expected it to do.